modules/islandora_binary_object_zip_importer/includes/importer.inc

```diff
@@ -208,6 +208,29 @@ class BinaryObjectZipBatchImportObject extends ZipBatchImportObject {
       $datastream['label'] = $this->getLabel($datastream['dsid'], $datastream);
     }
 
+    // See if the user uploaded DC, if not create it now.
+    if (!isset($to_return['DC'])) {
+      $obtained_dc = $this->getDC();
+      if ($obtained_dc) {
+        file_unmanaged_save_data($obtained_dc, 'public://DC record.xml');
```

Comment:
... Have to use the return from this, as there's a good chance the URI actually used will have a whatever numbers suffix'd on, right?